pdf/renderer.go

```diff
@@ -901,15 +892,10 @@ func (w *pdfPageWriter) WriteText(TJ ...interface{}) {
 			var rPrev rune
 			for j, r := range val {
 				if i < j {
-					i0, err0 := w.font.sfnt.GlyphIndex(&sfntBuffer, rPrev)
-					i1, err1 := w.font.sfnt.GlyphIndex(&sfntBuffer, r)
-					if err0 == nil && err1 == nil {
-						kern, err := w.font.sfnt.Kern(&sfntBuffer, i0, i1, toI26_6(units), font.HintingNone)
-						if err == nil && kern != 0.0 {
-							write(val[i:j])
-							fmt.Fprintf(w, " %d", -int(fromI26_6(kern)*1000.0/units+0.5))
-							i = j
-						}
+					if kern, err := w.font.Kerning(rPrev, r, 1000); err == nil && kern != 0 {
```

Comment:
Nitpick, but I like to write `kern != 0.0` to emphasize it's a float not an int (matter of taste I guess!)

Reply:
Ok (I usually `0`, and I saw that golang.org/x/image/font/sfnt was also using `return 0, err` on `Kerning`)